src/app/video-games/detail/components/video-game-detail/video-game-detail.component.html

```diff
@@ -1,3 +1,16 @@
 <h2>{{videoGame?.title}}</h2>
 <p>{{videoGame?.description}}</p>
-<app-youtube-video [url]="videoGame.youtubeUrl"></app-youtube-video>
\ No newline at end of file
+<app-youtube-video [url]="videoGame.youtubeUrl"></app-youtube-video>
+<div class="btn-group">
+  <button class="btn go-back-btn" (click)="goBack();">
+    <span class="fa fa-arrow-circle-left"></span> Go Back
+  </button>
+
+  <button (click)="toggleFavorite();" *ngIf="!videoGame.favorite" class="btn favorite-btn">
+    <span class="fa fa-heart"></span> Favorite
+  </button>
+
+  <button (click)="toggleFavorite();" *ngIf="videoGame.favorite" class="btn favorite-btn active">
+    <span class="fa fa-check"></span> Favorited
+  </button>
+</div>
```

Comment:
The favourite should go into a `app-favourite-toggle` component

Further, the template can be refactored - notice that the two button definitions are nearly identical.